cmd/webhook/main.go

```diff
@@ -79,32 +79,37 @@ func main() {
 	}
 
 	options := webhook.ControllerOptions{
-		ServiceName:    "tekton-pipelines-webhook",
-		DeploymentName: "tekton-pipelines-webhook",
-		Namespace:      system.GetNamespace(),
-		Port:           8443,
-		SecretName:     "webhook-certs",
-		WebhookName:    "webhook.tekton.dev",
+		ServiceName:                     "tekton-pipelines-webhook",
+		DeploymentName:                  "tekton-pipelines-webhook",
+		Namespace:                       system.GetNamespace(),
+		Port:                            8443,
+		SecretName:                      "webhook-certs",
+		WebhookName:                     "webhook.tekton.dev",
+		ResourceAdmissionControllerPath: "/",
 	}
-	//TODO add validations here
```

Comment:
Do we need to lose this comment?

Reply:
So I removed it because we are already doing validation in the webhook, so I felt it wasn't necessary anymore.